templates/common/git/git_test.go

```diff
@@ -0,0 +1,230 @@
+// Copyright 2023 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package git
+
+import (
+	"context"
+	"errors"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/abcxyz/pkg/testutil"
+	"golang.org/x/exp/slices"
+)
+
+// To actually run the tests in this file, you'll need to set this environment
+// variable.
+//
+// For example:
+//
+//	$ ABC_TEST_NON_HERMETIC=true go test ./...
+const envName = "ABC_TEST_NON_HERMETIC"
+
+func skipUnlessEnvEnabled(t *testing.T) {
```

Comment:
Is this for manual testing only or there is plan to run this in CI?

Reply:
Currently just for manual testing. No plans to automate yet.